.github/workflows/build_notarized.yml

```diff
@@ -222,13 +225,6 @@ jobs:
         name: DuckDuckGo-${{ env.release-type }}-dSYM-${{ env.app-version }}
         path: ${{ github.workspace }}/release/${{ steps.set-outputs.outputs.dsym-name }}
 
-    - name: Extract Asana Task ID
```

Comment:
This had no effect, as it turned out. The output wasn't used in any further step, nor is it defined as one of the outputs of the entire job.

This job is optionally followed by create-dmg job which calls that action on its own.